### DISCUSSION TITLE
Trying to upload a file from Supabase UI: The object exceeded the maximum allowed size

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#13028] [Questions]
Hi! I'm trying to upload a file from Supabase UI, and getting the following error:

> The object exceeded the maximum allowed size

How can I modify the maximum allowed upload size?

**Comment by GaryAustin1** [comment]
![image](https://user-images.githubusercontent.com/54564956/225345151-5a47b60e-39c3-4e60-b175-bb1f64c4f70f.png)
